static/app/components/events/autofix/autofixBanner.spec.tsx

```diff
@@ -1,83 +1,96 @@
+import {useIsSentryEmployee} from 'sentry/utils/useIsSentryEmployee';
+
+jest.mock('sentry/utils/useIsSentryEmployee');
+jest.mock('sentry/actionCreators/modal');
+
+const mockGroup = GroupFixture();
+const mockProject = ProjectFixture();
+const mockEvent = EventFixture();
 
 describe('AutofixBanner', () => {
-  afterEach(() => {
-    jest.resetAllMocks();
+  beforeEach(() => {
+    jest.clearAllMocks();
+    (useIsSentryEmployee as jest.Mock).mockReturnValue(false);
   });
 
-  const defaultProps = {
-    groupId: '1',
-    hasSuccessfulSetup: true,
-    triggerAutofix: jest.fn(),
-  };
-
-  it('shows PII check for sentry employee users', () => {
-    mockIsSentryEmployee(true);
-
-    render(<AutofixBanner {...defaultProps} projectId="1" />);
-    expect(
-      screen.getByText(
-        'By clicking the button above, you confirm that there is no PII in this event.'
-      )
-    ).toBeInTheDocument();
+  it('renders the banner with "Set up Autofix" button when setup is not done', () => {
+    render(
+      <AutofixBanner
+        group={mockGroup}
+        project={mockProject}
+        event={mockEvent}
+        hasSuccessfulSetup={false}
+      />
+    );
+
+    expect(screen.getByText('Try Autofix')).toBeInTheDocument();
+    expect(screen.getByRole('button', {name: /Set up Autofix/i})).toBeInTheDocument();
   });
 
-  it('does not show PII check for non sentry employee users', () => {
-    mockIsSentryEmployee(false);
+  it('renders the banner with "Run Autofix" button when setup is done', () => {
+    render(
+      <AutofixBanner
+        group={mockGroup}
+        project={mockProject}
+        event={mockEvent}
+        hasSuccessfulSetup
+      />
+    );
 
-    render(<AutofixBanner {...defaultProps} projectId="1" />);
-    expect(
-      screen.queryByText(
-        'By clicking the button above, you confirm that there is no PII in this event.'
-      )
-    ).not.toBeInTheDocument();
+    expect(screen.getByText(/Try Autofix/i)).toBeInTheDocument();
+    expect(screen.getByRole('button', {name: /Run Autofix/i})).toBeInTheDocument();
   });
 
-  it('can run without instructions', async () => {
-    const mockTriggerAutofix = jest.fn();
+  it('opens the AutofixSetupModal when "Set up Autofix" is clicked', () => {
+    const mockOpenModal = jest.fn();
+    (openModal as jest.Mock).mockImplementation(mockOpenModal);
 
     render(
       <AutofixBanner
-        {...defaultProps}
-        triggerAutofix={mockTriggerAutofix}
-        projectId="1"
+        group={mockGroup}
+        project={mockProject}
+        event={mockEvent}
+        hasSuccessfulSetup={false}
       />
     );
-    renderGlobalModal();
 
-    await userEvent.click(screen.getByRole('button', {name: 'Get root causes'}));
-    expect(mockTriggerAutofix).toHaveBeenCalledWith('');
+    screen.getByRole('button', {name: /Set up Autofix/i}).click();
+    expect(openModal).toHaveBeenCalled();
```

Comment:
if its throwing an act warning can add a waitFor here
```suggestion
    await waitFor(() => expect(openModal).toHaveBeenCalled())
```